### PR TITLE
Update to laravel 5.1.19

### PR DESCRIPTION
**Pull request by bramdevries**
### Changed
- Upgraded to Laravel 5.1.19

---

Policies yo

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "php": ">=5.5.9",
         "anahkiasen/arrounded": "^0.6",
         "bugsnag/bugsnag-laravel": "^1.4",
-        "laravel/framework": "5.1.*",
+        "laravel/framework": "^5.1",
```
Comment (hannesvdvreken) [inline composer.json:11]: I think it's more safe to write `5.1.*`, cause it's not following semver

Comment (bramdevries) [inline composer.json:11]: Thing is, if I left it like that it only installed upto `5.1.3`

Comment (hannesvdvreken) [inline composer.json:11]: Because of the lock file? Didn't `composer update laravel/framework` update the lock file?

Comment (bramdevries) [inline composer.json:11]: It updated to `5.1.3`

Comment (hannesvdvreken) [inline composer.json:11]: Weird. You can manually change to `5.1.*` and run `composer update --lock`.

**Comment by hannesvdvreken** [comment]
:ship:

**Pull request merged**